architecture/networking/routes.adoc

```diff
@@ -597,6 +597,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`*router.openshift.io/haproxy.health.check.interval*`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`*haproxy.router.openshift.io/ip_whitelist*` | Sets a xref:whitelist[whitelist] for the route. |
 |`*haproxy.router.openshift.io/hsts_header*` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`*router.openshift.io/cookie-same-site=Strict*` | Sets `SameSite` attribute to `Strict`. This ensures that the cookie will not be sent along with requests initiated by third party websites. |
```

Comment:
the annotation name is `router.openshift.io/cookie-same-site` and `Strict` is just one of the accepted value, and another value is `Lax`.
so please follow https://docs.openshift.com/container-platform/4.9/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration to update this part.